src/bindings/swig/python2/tests/testpy2_key.py

```diff
@@ -1,35 +1,48 @@
-from __future__ import print_function
 import kdb, unittest
 
 class Key(unittest.TestCase):
 	def setUp(self):
-		self.key = kdb.Key("user/key",
-			kdb.KEY_NAME,    "user/foo/bar",
-			kdb.KEY_VALUE,   "value",
-			kdb.KEY_OWNER,   "myowner",
-			kdb.KEY_COMMENT, "mycomment",
-			kdb.KEY_UID,     "123",
-			kdb.KEY_GID,     456,
-			kdb.KEY_MODE,    0o644,
-			kdb.KEY_ATIME,   123,
-			kdb.KEY_MTIME,   "456",
-			kdb.KEY_CTIME,   789,
-			kdb.KEY_DIR,
-			kdb.KEY_META,    "by", "manuel",
-			kdb.KEY_NULL
+		self.key = kdb.Key("user/foo/bar",
+			kdb.KEY_VALUE, "value",
+			kdb.KEY_META,  "by",    "manuel",
+			kdb.KEY_META,  "owner", "myowner"
 			)
 
 		self.bkey = kdb.Key("system/bkey",
-			kdb.KEY_BINARY,
-			kdb.KEY_VALUE,   "bvalue\0\0",
+			kdb.KEY_FLAGS, kdb.KEY_BINARY,
+			kdb.KEY_VALUE, b"bvalue\0\0",
```

Comment:
yeah. I copied the line from the python3 test file. The b-prefix gets ignored in python2.